src/drive/web/modules/drive/Toolbar/components/DocumentTypeData.js

```diff
@@ -0,0 +1,263 @@
+    subClassification: 'car',
+    label: 'driver_license'
+  },
+  {
+    id: '7',
+    classification: 'contract',
+    subject: 'weeding',
+    label: 'weeding'
+  },
+  {
+    id: '8',
+    classification: 'contract',
+    subject: 'pacs',
+    label: 'pacs'
+  },
+  {
+    id: '9',
+    classification: 'contract',
+    subject: 'divorce',
+    label: 'divorce'
+  },
+  {
+    id: '10',
+    classification: 'certificate',
+    subject: 'family',
+    label: 'large_family_card'
+  },
+  {
+    id: '11',
+    classification: 'certificate',
+    subject: 'family',
+    label: 'caf'
+  },
+  {
+    id: '12',
+    classification: 'diploma',
+    label: 'diploma'
+  },
+  {
+    id: '13',
+    classification: 'contract',
+    categorie: 'work',
+    label: 'work_contract'
+  },
+  {
+    id: '14',
+    classification: 'payslip',
+    label: 'pay_sheet'
+  },
+  {
+    id: '15',
+    classification: 'payslip',
+    label: 'unemployment_benefit'
+  },
+  {
+    id: '16',
+    classification: 'payslip',
+    label: 'pension'
+  },
+  {
+    id: '17',
+    classification: 'payslip',
+    label: 'other_revenue'
+  },
+  {
+    id: '18',
+    classification: 'office',
+    categorie: 'study',
+    label: 'gradebook'
+  },
+  {
+    id: '19',
+    classification: 'office',
+    label: 'health_book'
+  },
+  {
+    id: '20',
+    classification: 'certificate',
+    label: 'insurance_card'
+  },
+  {
+    id: '21',
+    classification: 'report',
+    categorie: 'health',
+    label: 'prescription'
+  },
+  {
+    id: '22',
+    classification: 'invoicing',
+    categorie: 'health',
+    label: 'health_invoice'
+  },
+  {
+    id: '23',
+    classification: 'certificate',
+    subject: 'car',
+    label: 'registration'
+  },
+  {
+    id: '24',
+    classification: 'certificate',
+    subject: 'car',
+    label: 'car_insurance'
+  },
+  {
+    id: '25',
+    classification: 'invoicing',
+    categorie: 'car',
+    label: 'mechanic_invoice'
+  },
+  {
+    id: '26',
+    classification: 'invoicing',
+    categorie: 'car',
+    label: 'transport_invoice'
+  },
+  {
+    id: '27',
+    classification: 'invoicing',
+    categorie: 'phone',
+    label: 'phone_invoice'
+  },
+  {
+    id: '28',
+    classification: 'invoicing',
+    categorie: 'isp',
+    label: 'isp_invoice'
+  },
+  {
+    id: '29',
+    classification: 'invoicing',
+    categorie: 'energy',
+    label: 'energy_invoice'
+  },
+  {
+    id: '30',
+    classification: 'invoicing',
+    categorie: 'web',
+    label: 'web_service_invoice'
+  },
+  {
+    id: '31',
+    classification: 'contract',
+    subject: 'house',
+    label: 'lease'
+  },
+  {
+    id: '32',
+    classification: 'contract',
+    subject: 'home',
+    label: 'rent_receipt'
+  },
+  {
+    id: '33',
+    classification: 'contract',
+    subject: 'house',
+    label: 'house_insurance'
+  }
+]
+export const categories = [
+  {
+    id: 'cat1',
+    label: 'identity',
+    icon: 'people',
+    file_type_ids: [1, 2, 3, 4, 5, 6]
```

Comment:
The ids are numbers here, whereas in the items they are strings, wouldn't it be better to use the same type?